docs/tutorials/prerequisites/install-graphviz.md

```diff
@@ -0,0 +1,9 @@
+## Install Graphviz
+#### On Ubuntu
+```
+sudo apt-get install graphviz
```

Comment:
Nope. With just the pip install, I get this error: https://github.com/dmlc/mxnet/pull/6617#discussion_r121017300